src/elements/content-sidebar/activity-feed/annotation-thread/AnnotationThreadCreate.js

```diff
@@ -36,16 +43,21 @@ const AnnotationThreadCreate = ({
     target,
 }: Props) => {
     const [isPending, setIsPending] = React.useState(false);
+    const events = useAnnotatorEvents({ eventEmitter });
 
     const handleCreate = (text: string): void => {
         const { id, permissions = {}, file_version = {} } = file;
+        const requestId = uniqueId('annotation_');
         setIsPending(true);
+        events.emitAddAnnotationStartEvent({ text }, requestId);
```

Comment:
I thought we made `requestId` optional? It's actually coming from Redux in box-annotations, so maybe it's not necessary? I suspect it has a different format, as well.